rdt/hyper_transformer.py

```diff
@@ -12,6 +12,18 @@
     get_default_transformer, get_transformer_instance, get_transformers_by_type)
 
 
+class Config(dict):
```

Comment:
Now that we have this class I'm wondering if it makes sense to move the `field_transformers`, `_provided_field_transformers`, `field_sdtypes` and `_provided_field_sdtypes` to this class. Then we could make the config an attribute of the `HyperTransformer` and add methods to this class for updating the config. For example, if the user sets something, it will update both dictionaries appropriately. Like a private `update_field_transformers(self, user_provided=False)` and `update_field_sdtypes(self, user_provided=False)` method that either only update the main dictionaries or both if necessary.

It's up to you if you want to do that extra work. I think the benefit would be that it would be less likely for anyone to forget to update or clear a provided dictionary when updating the main one, but this isn't in the scope of the issue

Reply:
I opened an issue to address this in another `pr` : https://github.com/sdv-dev/RDT/issues/457